index.html

```diff
@@ -101,10 +114,16 @@ <h2>
 			Please get in touch!
 		</p>
 		<div class="social-menu">
-				<a class="button linkedin" href="https://www.linkedin.com/in/ellie-robinson-63326b133/" target="_blank"><i class="fab fa-linkedin"></i></a>
-				<a class="button github" href="https://github.com/ellie-r" target="_blank"><i class="fab fa-github"></i></a>
+				<a class="button linkedin" href="https://www.linkedin.com/in/ellie-robinson-63326b133/" target="_blank">
+					<i class="fab fa-linkedin"></i>
+				</a>
+				<a class="button github" href="https://github.com/ellie-r" target="_blank">
+					<i class="fab fa-github"></i>
+				</a>
 				<!-- want to eventually make this button copy my email address to their clipboard-->
-				<a class="button email" href="#" target="_blank"><i class="fas fa-envelope-square"></i></a>
+				<a class="button email" href="#" target="_blank">
```

Comment:
I should've commented on this the first time round but you should make it link to your email like this:
```
<a class="button email" href="mailto: abc@example.com" target="_blank"> 
```

This way it will open an email client to send an email to you. It is also best if you only have features that are fully finished when merging into master - if not, it can live on another branch until it is ready.